ui/src/app/pages/access/access-renewal.spec.tsx

```diff
@@ -1,544 +0,0 @@
-import * as React from 'react';
```

Comment:
these tests have important logic checks - how can we ensure that tests for the combined page cover these cases?

Reply:
Good call. I will verify that these tests are covered elsewhere.